test/SourceKit/SyntaxMapData/Inputs/syntaxmap.swift

```diff
@@ -56,10 +56,10 @@ func testArgumentLabels(in class: Int, _ case: (_ default: Int) -> Void) -> (in:
   return something ? result : (in: 2, "foo")
 }
 
-// https://bugs.swift.org/browse/SR-9576
+// https://github.com/apple/swift/issues/52023
 func someFunc(input :Int?, completion: () throws -> Void) rethrows {}
 
-// https://bugs.swift.org/browse/SR-9740
+// https://github.com/apple/swift/issues/52171
```

Comment:
This change broke the test due to a shift in the byte offsets of subsequent syntactic elements, so I had to update the response file too.